### PR TITLE
blog: add v0.17.3.2 CLI and GUI release notes

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

File: _posts/2022-04-29-monero-GUI-0.17.3.2-released.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+The complete list of changes is [available on GitHub](https://github.com/monero-project/monero-gui/compare/v0.17.3.1...v0.17.3.2), along with [the source code](https://github.com/monero-project/monero-gui/tree/v0.17.3.2).
+
+# Contributors for this Release
+
+This release was the direct result of 22 who worked, largely unpaid and altruistically, to put out 76 commits containing 31625 new lines of code. We'd like to thank them very much for their time and effort. In no particular order they are:
+
+- selsta
+- luigi1111
+- rating89us
+- Monero-Weblate
```
Comment (erciccione) [inline _posts/2022-04-29-monero-GUI-0.17.3.2-released.md:42]: not a contributor, but you could add @netrik182, who is now managin that account

**Review by erciccione — COMMENTED** [review]
just a minor comment

File: _posts/2022-04-29-monero-GUI-0.17.3.2-released.md (hunk 1 above)
Comment (selsta) [reply _posts/2022-04-29-monero-GUI-0.17.3.2-released.md]: done

**Comment by binaryFate** [comment]
Hashes checked

**Pull request merged**